Xamarin.Forms.Platform.iOS/CollectionView/CollectionViewController.cs

```diff
@@ -12,6 +12,7 @@ internal class CollectionViewController : UICollectionViewController
 		readonly ItemsView _itemsView;
 		readonly ItemsViewLayout _layout;
 		bool _initialConstraintsSet;
+		int _previousCount = -1;
```

Comment:
do we need to reset this to -1 when the collection is cleared ?

Reply:
No. The -1 is only important on the very first run of `GetItemsCount`, when we are first setting up the CollectionView. We don't want the size update to run at that point because it will already be running as a result of the bounds of the CollectionView changing (to whatever the initial size is). After that, we can rely on the actual count.

Reply:
Could we not change this to a bool? like `hasRunCellLayout`. Then, in the `GetItemsCount` method, we check `if (!hasRunCellLayout && count > 0) hasRunCellLayout = true; ...`

Just like in `ItemsViewLayout` with `_needCellSizeUpdate`.